src/ov-brain.js

```diff
@@ -32,9 +32,9 @@ export default class OneViewBrain {
       if (res && res.members) {
         for (var i = 0; i < res.members.length; i++) {
           const sh = res.members[i];
-          Lexer.addNamedDevice(sh.name, sh.uri, 'name');
-          Lexer.addNamedDevice(sh.serialNumber, sh.uri, 'serialNumber');
-          robot.brain.set("__hpe__" + sh.uri, {uri:sh.uri, keys:{name:sh.name, serialNumber: sh.serialNumber}});
+          Lexer.addNamedDevice(sh.name, sh.uri, 'name', sh.hyperlink, robot);
```

Comment:
Why are you passing the robot here, is it needed by the addNamedDevice function?  Same question below.